src/rt-app.c

```diff
@@ -64,7 +64,6 @@ static pthread_mutex_t joining_mutex;
 static pthread_mutex_t fork_mutex;
 
 static ftrace_data_t ft_data = {
-	.debugfs = "/sys/kernel/debug",
```

Comment:
Could you update _ftrace_data_t and remove the now unused field debugfs ?
typedef struct _ftrace_data_t {
	char *debugfs;
	int trace_fd;
	int marker_fd;
} ftrace_data_t;

Also the trace_fd doesn't seem to be used anywhere which let the struct with only marker_fd. could be good to move it in opt struct as well

Reply:
@vingu-linaro thanks :) I will rebase with trace_fd removed in the first patch and I'll move marker_fd out. Incidentally, would you mind if I squash the patches together?

Reply:
I'm fine if you squash patches together

Reply:
Cool